### DISCUSSION TITLE
generate services and schema from a DB with tables feathers Dove

### DISCUSSION THREAD

**Discussion by haniyasin** [feathersjs/feathers#2804] [Q&A] [answered]
is there a way for a lazy indie developer to get the services/schema auto generated from a legacy DB with tables 
if not a hint is appreciated

**Comment by daffl** [comment, marked as answer]
Not officially at the moment but since JSON schema is a standard a tool that e.g. converts SQL table definitions to a JSON schema like https://github.com/duartealexf/sql-ddl-to-json-schema can probably help.
